package.json

```diff
@@ -45,7 +46,8 @@
     "typedoc": "^0.9.0"
   },
   "dependencies": {
-    "lodash": "^4.17.4",
+    "lodash": "^4.17.5",
```

Comment:
Both are needed, I only had to add these because I was facing errors in running `npm run develop`. Right now lodash imports are untouched. At some point I would change those imports to use `lodash-es`. With that I tihnk we can eventually get treeshaking benefit.

Reply:
Alright